source/content.js

```diff
@@ -16,6 +16,7 @@ import addFileCopyButton from './features/copy-file';
 // - import copyMarkdown from './features/copy-markdown';
 import linkifyCode from './features/linkify-urls-in-code';
 import autoLoadMoreNews from './features/auto-load-more-news';
+import autoLoadComntributionActivities from './features/auto-load-contribution-activities';
```

Comment:
Typo here in `autoLoadContributionActivities` :)

Reply:
:see_no_evil: Thanks for fetching that